.github/workflows/contrib-tests.yml

```diff
@@ -85,7 +85,7 @@ jobs:
       - name: Test Compression
         if: matrix.python-version != '3.10'
```

Comment:
Is this necessary?

Reply:
Yes, in line 53 of retrieval chat, the whole contrib folder is run on version==3.10, so we added this condition to avoid running it again. As I pointed out here: https://github.com/microsoft/autogen/pull/555#discussion_r1382775149. I think there is a pontential issue with 
`coverage run -a -m pytest test/test_retrieve_utils.py test/agentchat/contrib` being run in `RetrieveChatTest`, while dependencies for other contrib agents are not installed.

Reply:
@thinkall could you confirm this problem?